dungeon/src/contrib/crafting/Crafting.java

```diff
@@ -74,66 +79,42 @@ public static void clearRecipes() {
     RECIPES.clear();
   }
 
+  // See https://stackoverflow.com/a/28985785
+  public static List<URL> getResources(final String path) throws IOException {
```

Comment:
Sollte `private` sein - oder andernfalls einen doc string haben?

Reply:
@tgrothe du stellst immer so fragen - das moped kommt doch von dir?! du musst doch wissen, ob das in die öffentliche api mit rein muss oder ob das ne hilfsmethode ist? wenn public, dann javadoc.

Reply:
ok, dann ändere ich das noch.